charts/fluentbit/templates/clusterrole.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "calyptia-fluent-bit.fullname" . }}
+  labels:
+    {{- include "calyptia-fluent-bit.labels" . | nindent 4 }}
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - namespaces
+      - pods
+      {{- if .Values.rbac.nodeAccess }}
+      - nodes
+      - nodes/proxy
+      {{- end }}
+    verbs:
+      - get
+      - list
+      - watch
```

let's add a line at the end
```suggestion
      - watch

```